btrfsmaintenance-functions

```diff
@@ -72,7 +72,7 @@ check_balance_running() {
 #
 # check if filesystem is a btrfs
 is_btrfs() {
-	local FS=$(stat -f --format=%T "$1")
+	local FS=$(stat -f --format=%T $(df --output=target "$1")|tail -n1)
```

Comment:
Was this line suppose to be this?
```
local FS=$(stat -f --format=%T $(df --output=target "$1"|tail -n1))
```

Because I ran the command...
```
stat -f --format=%T $(df --output=target /dev/mapper/vg0-root)|tail -n1
```
and got some stat errors due to the `Mounted on` text that `df` prints.

Reply:
Opps bracket it is the wrong place.. Will fix and submit another pull. My bad.